docker/install/ubuntu_install_java.sh

```diff
@@ -20,5 +20,20 @@ set -o errexit -o nounset
 set -o pipefail
 
 apt-get update && apt-get install -y openjdk-8-jdk maven
-test -d "/usr/lib/jvm/java-8-openjdk-amd64/jre"
-echo "export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64/jre" >> /etc/profile
+arch=`uname -m`
+jre_arch="unknown"
+case $arch in
+    'x86_64')
+	jre_arch="amd64"
+	;;
+    'aarch64')
+	jre_arch="arm64"
+	;;
+    default)
+	echo "Unknown architecture output of uname -m"
+	exit 8
```

Comment:
Is there significance in the value 8? If not perhaps better to stick with 1 (because otherwise in 2 years time we will be scratching our heads tryhing to figure out whether or not the 8 was signficant ;-)